localstack-core/localstack/services/kms/provider.py

```diff
@@ -368,6 +370,9 @@ def _is_rsa_spec(key_spec: str) -> bool:
     #
 
     @handler("CreateKey", expand=False)
+    @kms_feature.Provisioning
+    @kms_api.CreateKey
+    @ApiLimitation(limitation="Status 'Updating' is not supported.")
```

Comment:
Both, `@kms_api.CreateKey` and `@ApiLimitation` are used as an example to track api-specific limitations. 
The `playground.py` script only considers the `ApiLimitation`, while in the `playgrond_kms_api_limitation.py` we collect the data from the former decorator.